src/editor/utils/elementinterface.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import getDataFromElement from '@ckeditor/ckeditor5-utils/src/dom/getdatafromelement';
+import setDataInElement from '@ckeditor/ckeditor5-utils/src/dom/setdatainelement';
+
+/**
+ * @module core/editor/utils/elementinterface
+ */
+
+/**
+ * Interface provides method for setting and getting data from/to element on which editor has been initialized.
+ *
+ * @mixin ElementInterface
+ * @implements module:core/editor/utils/elementinterface~Element
+ */
+const ElementInterface = {
```

Comment:
All the same notes as in case of the `EditorDataApi`. Here we could have:

* `EditorElementMixin` mixin
* `EditorElement` interface

But, just like with the "data" stuff, I don't like the fact that editor doesn't become an editor element but it gets an API of such type, so (however I don't like the length of this) the following seems more correct:

* `EditorElementApiMixin`
* `EditorElementApi`

PS. API ~= Interface, but I use API because we'd end up with `EditorElementInterfaceMixin` and `EditorElementInterface`. Unless... we'd decide that we can omit `Interface` suffix, but then we end up with an also ugly `EditorElement` interface.

In other words, I chose "API" because it reads well: "My editor implements the editor element API". While this makes no sense at all: "My editor implements editor element".